store-cli.go

```diff
@@ -82,6 +98,11 @@ func makeURL(storeType, scope, key string) (*url.URL, error) {
 }
 
 func get(storeType, scope, key string) error {
+	skip := skipCache(storeType, scope, "get")
+	if skip == true {
```

Comment:
`if skip {`?

Reply:
fixed. Thought you need to be explicit in go